build.gradle

```diff
@@ -10,11 +15,11 @@ repositories {
 }
 
 dependencies {
-    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.7.2'
-    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.7.2'
-
+    implementation group:'org.xerial', name:'sqlite-jdbc', version:'3.8.11.2'
```

Comment:
You are using the `sqlite-jdbc` version of the 2015 year. Can you update it?

Reply:
fixed